File: frontend/src/types/MultiplayerGame.ts
export type MultiplayerGame = {
  id?: string;
  playerIds: string[];
  difficulty: number;
  streakToWin: number;
  winnerId: string[];
  loserIds: string[];
  wonInMilliseconds: number;
  totalPlayers: number;
  createdAt?: string;
};

File: backend/src/main/java/de/neuefische/paulkreft/backend/game/multiplayer/model/MultiplayerGame.java
package de.neuefische.paulkreft.backend.game.multiplayer.model;

import org.springframework.data.annotation.Id;

import java.time.Instant;
import java.util.List;

public record MultiplayerGame(
        @Id
        String id,
        List<String> playerIds,
        int difficulty,
        Integer streakToWin,
        List<String> winnerIds,
        List<String> loserIds,
        Integer wonInMilliseconds,
        int totalPlayers,
        Instant createdAt
) {
}
